Shuttlerock/actions
# [STUDIO-586] Add the `has-issues` label when a PR has a failed check

## Add the `has-issues` label when a PR has a failed check

[Jira Tech task](https://shuttlerock.atlassian.net/browse/STUDIO-586)

## How to test the PR

`act --job check_run_completed_action --eventpath src/tests/fixtures/check-run-failed-payload.json`

It should add the `has-issues` label, move the Jira issue to `Has issues`, and send a Slack message to the Jira assignee.

## Deployment Notes

None
